tests/sync/test_sync.py

```diff
@@ -99,7 +99,7 @@ def test_sync_gradually_add_nodes(init_session, setup_bootstrap, save_log_on_exi
 
     start = time.time()
     sleep = 30  # seconds
-    num_iter = 20  # total of 5 minutes
+    num_iter = 25  # total of 5 minutes
```

Comment:
intentional?

Reply:
Yes, Almog said the sync test was a bit broken that we should have given it a bit more time